File: yggdrasil/examples/composite_function/src/model_function.py
import numpy as np


def model_function(a, b, c):
    out = np.zeros(3, 'float64')
    for i in range(3):
        if a:
            out[i] = b * (i ** c["c1"])
        else:
            out[i] = b * (i ** c["c2"])
    return out

File: yggdrasil/examples/composite_function/src/model_function.R
model_function <- function(a, b, c) {
  out <- double(3)
  for (i in 1L:3) {
    if (a) {
      out[[i]] <- b * (i-1)^c[["c1"]]
    } else {
      out[[i]] <- b * (i-1)^c[["c2"]]
    }
  }
  return(out)
}
